src/rgw/rgw_sal_store.h

```diff
@@ -404,9 +404,22 @@ class StoreZone : public Zone {
     virtual ~StoreZone() = default;
 };
 
-class StoreLuaScriptManager : public LuaScriptManager {
+class StoreLuaManager : public LuaManager {
 public:
-  virtual ~StoreLuaScriptManager() = default;
+  virtual ~StoreLuaManager() = default;
+
+  /** Get a script named with the given key from the backing store */
+  virtual int get_script(const DoutPrefixProvider* dpp, optional_yield y, const std::string& key, std::string& script) { return -ENOENT; }
```

Comment:
if these stubs are implemented here, we can probably avoid them in motr dbstore?

Reply:
okay. My C++ is still pretty rusty. ;-)